tests/auto_indent_test.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import print_function
+
+import filecmp
+import os
+import sys
+from shutil import copyfile
+
+import pytest
+
+from pre_commit_hooks.auto_indent import main
+from pre_commit_hooks.auto_indent import RETURN_CODE
+from testing.util import get_resource_path
+
+
+def _has_ast_arg_lineno_support():
+    """Checks if AST has all the info necessary to auto indent.
+
+    Some Python implementations removed line numbers from AST arg objects and
+    as a result we can't automatically format function calls.
+    """
+    if sys.version_info.major == 3 and sys.version_info.minor == 3:
+        return False
+    elif sys.version_info.major == 3 and sys.version_info.minor == 2:
+        return False
+    else:
+        return True
+
+
+@pytest.mark.parametrize(
+    ('infile', 'expected_rc'),
+    (
+        ('file_with_acceptable_indentation', RETURN_CODE['no_change']),
```

Comment:
in general don't put logic in tests -- I'd much rather see the actual value here than a constant

Here's a good article to read about this subject: http://googletesting.blogspot.com/2014/07/testing-on-toilet-dont-put-logic-in.html